projects/decoder_tests/Cargo.lock

```diff
@@ -11,4 +11,4 @@ dependencies = [
 
 [[package]]
 name = "xrpl-std"
-version = "0.0.1"
+version = "0.5.0-devnet5"
```

Comment:
If we're merging into `main` then we probably want to update this dependency (though maybe that's a different PR?)

Reply:
This is auto-generated from here:
https://github.com/ripple/craft/blob/aef1f5462c2e0f9bd7e7dd63cab051e29f91096e/xrpl-std/Cargo.toml#L3
I will update that file.

Reply:
I have a separate PR for this, let's defer to that one: https://github.com/ripple/craft/pull/142